controller/games.controller.js

```diff
@@ -0,0 +1,7 @@
+const { selectAllCategories } = require("../model/games.model");
+
```

Comment:
consider renaming this file to categories.controller and the same for your model. When time to move onto reviews, that would be reviews.controller. This keeps the functionality for the different parts of the database separate. (And much easer to debug)